packages/core/strapi/lib/core/app-configuration/load-config-file.js

```diff
@@ -30,6 +30,10 @@ const loadJSONFile = file => {
 const loadFile = file => {
   const ext = path.extname(file);
 
+  if(ext !== '.js' && ext !== '.json' && ext !== '.ts') {
+    throw new Error(`Remove an invalid config file: ${file}`);
```

Comment:
What we want instead is to ignore those files instead of throwing. 

The simplest approach I see is to change the default to return undefined and update this file like so:

`packages/core/strapi/lib/core/app-configuration/config-loader.js `

```diff
'use strict';

const path = require('path');
const fs = require('fs');
const loadFile = require('./load-config-file');

module.exports = dir => {
  if (!fs.existsSync(dir)) return {};

  return fs
    .readdirSync(dir, { withFileTypes: true })
    .filter(file => file.isFile())
    .reduce((acc, file) => {
      const key = path.basename(file.name, path.extname(file.name));

-      acc[key] = loadFile(path.resolve(dir, file.name));
+      acc[key] = loadFile(path.resolve(dir, file.name)) || acc[key];


      return acc;
    }, {});
};

```